File: Master/Year 1/Semester 1/AMSS/Proiect/app/src/main/java/ro/robertto/rescuepets/presentation/model/Message.java
package ro.robertto.rescuepets.presentation.model;

public class Message {
    private String message;
    private String senderId;
    private String messageId;
    private String imageUrl;

    public Message() {
        // Default constructor required for Firebase
    }

    public Message(String message, String senderId) {
        this.message = message;
        this.senderId = senderId;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getSenderId() {
        return senderId;
    }

    public void setSenderId(String senderId) {
        this.senderId = senderId;
    }

    public String getMessageId() {
        return messageId;
    }

    public void setMessageId(String messageId) {
        this.messageId = messageId;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }
}

File: Licenta/Anul 2/Semestrul 2/MDS/Proiect/EmergencyAppSignUp/app/src/main/java/com/lifeSavers/emergencyappsignup/model/Message.kt
package com.lifeSavers.emergencyappsignup.model

class Message {
    var messageId: String? = null
    var message: String? = null
    var senderId: String? = null
    var imageUrl: String? = null
    var timeStamp: Long = 0

    constructor()

    constructor(
        message: String?,
        senderId: String?,
        timeStamp: Long
    ) {
        this.message = message
        this.senderId = senderId
        this.timeStamp = timeStamp
    }

}